src/realm/sync/subscriptions.cpp

```diff
@@ -94,17 +94,12 @@ Timestamp Subscription::updated_at() const
     return m_updated_at;
 }
 
-bool Subscription::has_name() const
-{
-    return static_cast<bool>(m_name);
-}
-
-std::string_view Subscription::name() const
+util::Optional<std::string_view> Subscription::name() const
 {
```

Comment:
Blarg. If we used std::optional, I think you could just do `return m_name;`. Maybe someday..